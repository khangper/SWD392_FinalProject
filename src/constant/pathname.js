export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
