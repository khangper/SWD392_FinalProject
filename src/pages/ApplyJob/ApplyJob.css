.apply-job-icon {
  width: 10px;
  height: 10px;
  margin-top: 4.5px;
  margin-right: 10px;
}

.apply-job-back-to-career {
  position: relative;
  line-height: 20px;
  display: inline-block;
  min-width: 79px;
  white-space: nowrap;
}

.apply-job-back-to-career-link {
  align-self: stretch;
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
}

.apply-job-home {
  position: relative;
  line-height: 24px;
  color: #333;
  display: inline-block;
  min-width: 38px;
}

.apply-job-breadcrumb-items,
.apply-job-breadcrumb-items-1,
.apply-job-data-engineer {
  min-width: 6.1px;
  position: relative;
  line-height: 24px;
  display: inline-block;
}

.apply-job-career {
  position: relative;
  line-height: 24px;
  display: inline-block;
  color: #333;
  min-width: 54px;
  white-space: nowrap;
}

.apply-job-data-engineer {
  min-width: 104px;
  white-space: nowrap;
}

.apply-job-breadcrumb-list {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
  gap: 6.6px;
}

.apply-job-heading-1 {
  position: relative;
  font-size: 24px;
  line-height: 30px;
  font-weight: 550;
  color: #333;
  white-space: nowrap;
}

.apply-job-breadcrumb-content,
.apply-job-breadcrumb-title {
  display: flex;

  align-items: flex-start;
}

.apply-job-breadcrumb-title {
  padding: 0 20px 0 0;
  gap: 30px;
  color: rgba(33, 37, 41, 0.75);
  flex-direction: column;
  justify-content: flex-start;
}

.apply-job-breadcrumb-content {
  width: 100%;
  max-width: 100%;
  flex-direction: row;
  justify-content: space-between;
  padding: 20px 190px;
  height: 120px;
}
/*Apply For This Job*/

.apply-job-apply-for-button {
  display: flex;
  padding: 13px 15px;
  justify-content: center;
  align-items: center;
  border-radius: 3px;
  background-color: #F9BF4B;
  border: none;
  color: #fff;
  font-weight: bold;
  cursor: pointer;
}

.apply-job-apply-for-button:hover {
  background-color: #333;
}

.apply-job-horizontal-border,
.apply-job-horizontal-border-wrapper {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
  max-width: 100%;
}

.apply-job-horizontal-border {
  flex: 1;
  border-bottom: 1px solid #e5e5e5;
  padding: 0 0 30px;
}

.apply-job-horizontal-border-wrapper {
  align-self: stretch;
}

.apply-job-content,
.apply-job-content-1 {
  margin: 0;
  color: #686f7a;
}

.apply-job-content-container,
.apply-job-content-container-1 {
  position: relative;
  line-height: 26px;
}

.apply-job-content-container-1 {
  margin-left: 10px;
}

.apply-job-heading-41 {
  position: relative;
  font-size: 18px;
  line-height: 26px;
  font-weight: 550;
  color: #333;
  display: inline-block;
  max-width: 100%;
}

.apply-job-description {
  gap: 15px;
  display: flex;
  flex-direction: column;
  border-bottom: 1px solid #e5e5e5;
  padding: 0 0 30px;
}

.apply-job-application-parent,
.apply-job-application {
  display: flex;
  align-items: flex-start;
  max-width: 100%;
}

.apply-job-application-parent {
  flex-direction: column;
  justify-content: flex-start;
  gap: 30px;
  width: 780px;
  padding: 0 2.5px;
}

.apply-job-application {
  align-self: stretch;
  flex-direction: row;
  justify-content: center;
  padding-top: 40px;
  padding-bottom: 70px;
  box-sizing: border-box;
  background-color: #f7f7f7;
}

.apply-job-wrapper {
  display: flex;
  align-items: flex-start;
  justify-content: flex-start;
  max-width: 100%;
  flex-shrink: 0;
  text-align: left;
  font-size: 14px;
  color: #686f7a;
  flex: 1;
  flex-direction: column;
  box-sizing: border-box;
}

.apply-job-border-form {
  flex: 1;
  position: relative;
}
.apply-job-border-form input,
.apply-job-border-form textarea,
.apply-job-select .ant-select-selector {
  width: 100%;
  max-width: 100%;
  min-width: 100%;
  padding: 10px;
  background: #fff;
  border: 1px solid #e5e5e5 !important;
  outline: 0;
  color: rgba(0, 0, 0, 0.87);
  border-radius: 3px !important;
  box-shadow: inset 0px 1px 4px rgba(0, 0, 0, 0.02) !important;
}

.apply-job-select .ant-select-selector{
  font-size: 13px;
}

.apply-job-border-form input::placeholder,
.apply-job-border-form textarea::placeholder {
  color: lightgray;
}

.apply-job-form {
  display: flex;
  flex-direction: column;
  gap: 15px;
  padding: 10px 0;
}

.apply-job-heading-42 {
  position: relative;
  font-size: 14px;
  line-height: 26px;
  font-weight: 550;
  color: #333;
  display: inline-block;
  max-width: 100%;
}

.apply-job-submit-application,
.apply-job-links {
  width: 100%;
  gap: 20px;
  display: flex;
  flex-direction: column;
  border-bottom: 1px solid #e5e5e5;
  padding: 0 0 50px;
}

.apply-job-additional-infor {
  width: 100%;
  gap: 40px;
  display: flex;
  flex-direction: column;
}
.apply-job-cv-form {
  border: 1px solid #e5e5e5;
  position: relative;
  padding: 30px;
  text-align: center;
  cursor: pointer;
  width: 100%;
  background-color: #fff;
}

.apply-job-cv-form input {
  opacity: 0;
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  width: 100%;
  height: 100%;
  border-radius: 3px;
  cursor: pointer;
}

.apply-job-icon-upload {
  width: 30px;
  height: 30px;
}

.apply-job-drap-text h4 {
  font-size: 16px;
  color: #333;
}

.apply-job-submit-button {
  display: flex;
  padding: 13px 15px;
  justify-content: center;
  align-items: center;
  border-radius: 3px;
  background-color: #196c7e;
  border: none;
  color: #fff;
  font-weight: bold;
  cursor: pointer;
  width: 160px;
}

.apply-job-submit-button:hover {
  background-color: #333;
}
