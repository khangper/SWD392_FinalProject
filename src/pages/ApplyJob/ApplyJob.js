import React, { useState } from "react";
import "./ApplyJob.css";
import Header from "../../components/Header/Header";
import { MdKeyboardDoubleArrowLeft } from "react-icons/md";
import { MdCloudUpload } from "react-icons/md";
import Footer from "../../components/Footer/Footer";

const ApplyJob = () => {
  return (
    <div className="apply-job">
      <Header />
      <main className="apply-job-parent">
        <div className="apply-job-wrapper">
          <div className="apply-job-breadcrumb-content">
            <div className="apply-job-breadcrumb-title">
              <div className="apply-job-breadcrumb-list">
                <a href="/home" className="apply-job-home">
                  Home
                </a>
                <div className="apply-job-breadcrumb-items">/</div>
                <a href="/career" className="apply-job-career">
                  Career
                </a>
                <div className="apply-job-breadcrumb-items-1">/</div>
                <div className="apply-job-data-engineer">Data Engineer</div>
              </div>
              <h3 className="apply-job-heading-1">Data Engineer</h3>
            </div>
            <div className="apply-job-back-to-career-link">
              <MdKeyboardDoubleArrowLeft className="apply-job-icon" />
              <a href="/career" className="apply-job-back-to-career">
                Back to Career
              </a>
            </div>
          </div>

          <div className="apply-job-application">
            <div className="apply-job-application-parent">
              <div className="apply-job-horizontal-border-wrapper">
                <div className="apply-job-horizontal-border">
                  <button className="apply-job-apply-for-button">
                    Apply For This Job
                  </button>
                </div>
              </div>

              <div className="apply-job-description">
                <div className="apply-job-content-container">
                  <p className="apply-job-content">
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                    Curabitur volutpat maximus pellentesque. Integer sem enim,
                    luctus at nibh at, condimentum sagittis sapien. Sed tempus
                    ipsum erat, sit amet efficitur velit interdum eu. Vestibulum
                    hendrerit id dolor eu scelerisque. Phasellus ex dui,
                    consequat nec feugiat eu, dapibus eget ante. Sed sodales
                    interdum dui, at euismod mi feugiat hendrerit. Suspendisse
                    auctor libero in tempor mollis. Nulla et dolor velit.
                    Aliquam sit amet luctus quam.
                  </p>
                </div>

                <div className="apply-job-heading-41">What you'll be doing</div>

                <div className="apply-job-content-container">
                  <p className="apply-job-content">
                    Nam a egestas libero, eget eleifend turpis. Sed id ipsum a
                    ipsum aliquam laoreet sit amet sit amet nibh. Proin dapibus,
                    libero sed posuere rhoncus, orci mi cursus enim, at accumsan
                    eros massa lacinia mi. Nunc eget finibus felis, volutpat
                    malesuada sem. Aliquam ac nisl pellentesque, varius neque
                    sit amet, porttitor nunc. Nullam elit tellus, dapibus non
                    eleifend sed, hendrerit eget velit. Aliquam ut felis dictum,
                    tincidunt magna vitae, aliquam massa. In porttitor tristique
                    quam, non dignissim sapien pharetra ultrices. Cras non ante
                    non velit mollis mollis. Pellentesque habitant morbi
                    tristique senectus et netus et malesuada fames ac turpis
                    egestas. Quisque et bibendum urna, eget consequat sapien.
                    Integer sed condimentum nibh. Integer id neque tristique,
                    lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor
                    ac rutrum eget, volutpat a velit. Curabitur et enim quis
                    diam congue dictum et vitae dui. Nulla tortor orci, luctus a
                    pretium vel, ultrices porta nisl.
                  </p>
                </div>

                <div className="apply-job-content-container">
                  <p className="apply-job-content">
                    Etiam lobortis dictum tincidunt. Interdum et malesuada fames
                    ac ante ipsum primis in faucibus. Suspendisse ultricies
                    efficitur dui, suscipit tempus elit condimentum quis. Duis
                    sed vestibulum tortor, eget cursus odio.
                  </p>
                </div>

                <div className="apply-job-heading-41">
                  What we're looking for
                </div>

                <div className="apply-job-content-container-1">
                  <p className="apply-job-content-1">
                    1. Work closely with other data engineers to enhance
                    infrastructure, improve reliability, and strengthen database
                    performance
                  </p>
                  <p className="apply-job-content-1">
                    2. Collaborate with data scientists, product managers,
                    marketers and other data-driven internal stakeholders to
                    identify opportunities for better data access and usage
                  </p>
                  <p className="apply-job-content-1">
                    3. Contribute to the ongoing development of the data
                    warehouse ecosystem
                  </p>
                  <p className="apply-job-content-1">
                    4. Become the expert on all aspects of Edututs+'s data
                    reporting and analytics infrastructure
                  </p>
                  <p className="apply-job-content-1">
                    5. Develop and maintain data definitions for internally
                    available data sources
                  </p>
                </div>

                <div className="apply-job-heading-41">Who should apply?</div>

                <div className="apply-job-content-container">
                  <p className="apply-job-content">
                    Quisque et bibendum urna, eget consequat sapien. Integer sed
                    condimentum nibh. Integer id neque tristique, lobortis massa
                    ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum
                    eget, volutpat a velit. Curabitur et enim quis diam congue
                    dictum et vitae dui. Nulla tortor orci, luctus a pretium
                    vel, ultrices porta nisl.
                  </p>
                </div>

                <div className="apply-job-heading-41">What next?</div>

                <div className="apply-job-content-container">
                  <p className="apply-job-content">
                    Quisque et bibendum urna, eget consequat sapien. Integer sed
                    condimentum nibh. Integer id neque tristique, lobortis massa
                    ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum
                    eget, volutpat a velit. Curabitur et enim quis diam congue
                    dictum et vitae dui. Nulla tortor orci, luctus a pretium
                    vel, ultrices porta nisl.
                  </p>
                </div>
              </div>

              <div className="apply-job-submit-application">
                <div className="apply-job-heading-41">
                  Submit Your Application
                </div>
                <div className="apply-job-form">
                  <div className="apply-job-border-form">
                    <input
                      type="text"
                      placeholder="Full Name"
                      id="fullname"
                      name="fullname"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <input
                      type="email"
                      placeholder="Email Adress"
                      id="email"
                      name="email"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <input
                      type="number"
                      placeholder="Phone Number"
                      id="phonenumber"
                      name="phonenumber"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <select name="gender" defaultValue="">
                      <option value="">Gender</option>
                      <option value="male">Male</option>
                      <option value="female">Female</option>
                    </select>
                  </div>
                </div>

                <div className="apply-job-heading-42">Resume/CV*</div>
                <div className="apply-job-cv-form">
                  <input
                    type="file"
                    name="resume"
                    accept="application/pdf"
                    placeholder="Attach Resume/CV"
                  />
                  <div className="apply-job-drap-text">
                    <MdCloudUpload className="apply-job-icon-upload" />
                    <h4>Attach Resume/CV</h4>
                  </div>
                </div>
              </div>

              <div className="apply-job-links">
                <div className="apply-job-heading-41">Links</div>
                <div className="apply-job-form">
                  <div className="apply-job-border-form">
                    <input
                      type="url"
                      placeholder="LinkedIn URL"
                      id="linkedin"
                      name="linkedin"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <input
                      type="url"
                      placeholder="Twitter URL"
                      id="twitter"
                      name="twitter"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <input
                      type="url"
                      placeholder="Postfolio URL"
                      id="postfolio"
                      name="postfolio"
                    />
                  </div>
                  <div className="apply-job-border-form">
                    <input
                      type="url"
                      placeholder="Other Website URL"
                      id="otherweb"
                      name="otherweb"
                    />
                  </div>
                </div>
              </div>

              <div className="apply-job-additional-infor">
                <div className="apply-job-heading-41">Links</div>
                <div className="apply-job-border-form">
                  <textarea
                    name="infor"
                    id="infor"
                    placeholder="Add a cover letter or anything else you want to share"
                    rows={30}
                    cols={10}
                  ></textarea>
                </div>

                <button className="apply-job-submit-button" type="submit">
                  Submit Application
                </button>
              </div>
            </div>
          </div>
        </div>
      </main>
      <Footer/>
    </div>
  );
};

export default ApplyJob;
