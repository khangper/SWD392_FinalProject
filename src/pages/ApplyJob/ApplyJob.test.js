import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";
import ApplyJob from "../ApplyJob/ApplyJob";

describe("ApplyJob", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the ApplyJob component", async () => {
    const { container } = renderWithProviders(<ApplyJob />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});