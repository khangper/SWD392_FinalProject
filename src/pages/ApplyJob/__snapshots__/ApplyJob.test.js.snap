// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplyJob renders the ApplyJob component 1`] = `
<div>
  <div
    class="apply-job"
  >
    <header
      class="header"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <div
        class="profile"
      >
        <img
          alt=""
          class="header-avatar"
          src="profile-img.jpg"
        />
      </div>
    </header>
    <main
      class="apply-job-parent"
    >
      <div
        class="apply-job-wrapper"
      >
        <div
          class="apply-job-breadcrumb-content"
        >
          <div
            class="apply-job-breadcrumb-title"
          >
            <div
              class="apply-job-breadcrumb-list"
            >
              <a
                class="apply-job-home"
                href="/home"
              >
                Home
              </a>
              <div
                class="apply-job-breadcrumb-items"
              >
                /
              </div>
              <a
                class="apply-job-career"
                href="/career"
              >
                Career
              </a>
              <div
                class="apply-job-breadcrumb-items-1"
              >
                /
              </div>
              <div
                class="apply-job-data-engineer"
              >
                Data Engineer
              </div>
            </div>
            <h3
              class="apply-job-heading-1"
            >
              Data Engineer
            </h3>
          </div>
          <div
            class="apply-job-back-to-career-link"
          >
            <svg
              class="apply-job-icon"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M17.59 18 19 16.59 14.42 12 19 7.41 17.59 6l-6 6z"
              />
              <path
                d="m11 18 1.41-1.41L7.83 12l4.58-4.59L11 6l-6 6z"
              />
            </svg>
            <a
              class="apply-job-back-to-career"
              href="/career"
            >
              Back to Career
            </a>
          </div>
        </div>
        <div
          class="apply-job-application"
        >
          <div
            class="apply-job-application-parent"
          >
            <div
              class="apply-job-horizontal-border-wrapper"
            >
              <div
                class="apply-job-horizontal-border"
              >
                <button
                  class="apply-job-apply-for-button"
                >
                  Apply For This Job
                </button>
              </div>
            </div>
            <div
              class="apply-job-description"
            >
              <div
                class="apply-job-content-container"
              >
                <p
                  class="apply-job-content"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur volutpat maximus pellentesque. Integer sem enim, luctus at nibh at, condimentum sagittis sapien. Sed tempus ipsum erat, sit amet efficitur velit interdum eu. Vestibulum hendrerit id dolor eu scelerisque. Phasellus ex dui, consequat nec feugiat eu, dapibus eget ante. Sed sodales interdum dui, at euismod mi feugiat hendrerit. Suspendisse auctor libero in tempor mollis. Nulla et dolor velit. Aliquam sit amet luctus quam.
                </p>
              </div>
              <div
                class="apply-job-heading-41"
              >
                What you'll be doing
              </div>
              <div
                class="apply-job-content-container"
              >
                <p
                  class="apply-job-content"
                >
                  Nam a egestas libero, eget eleifend turpis. Sed id ipsum a ipsum aliquam laoreet sit amet sit amet nibh. Proin dapibus, libero sed posuere rhoncus, orci mi cursus enim, at accumsan eros massa lacinia mi. Nunc eget finibus felis, volutpat malesuada sem. Aliquam ac nisl pellentesque, varius neque sit amet, porttitor nunc. Nullam elit tellus, dapibus non eleifend sed, hendrerit eget velit. Aliquam ut felis dictum, tincidunt magna vitae, aliquam massa. In porttitor tristique quam, non dignissim sapien pharetra ultrices. Cras non ante non velit mollis mollis. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.
                </p>
              </div>
              <div
                class="apply-job-content-container"
              >
                <p
                  class="apply-job-content"
                >
                  Etiam lobortis dictum tincidunt. Interdum et malesuada fames ac ante ipsum primis in faucibus. Suspendisse ultricies efficitur dui, suscipit tempus elit condimentum quis. Duis sed vestibulum tortor, eget cursus odio.
                </p>
              </div>
              <div
                class="apply-job-heading-41"
              >
                What we're looking for
              </div>
              <div
                class="apply-job-content-container-1"
              >
                <p
                  class="apply-job-content-1"
                >
                  1. Work closely with other data engineers to enhance infrastructure, improve reliability, and strengthen database performance
                </p>
                <p
                  class="apply-job-content-1"
                >
                  2. Collaborate with data scientists, product managers, marketers and other data-driven internal stakeholders to identify opportunities for better data access and usage
                </p>
                <p
                  class="apply-job-content-1"
                >
                  3. Contribute to the ongoing development of the data warehouse ecosystem
                </p>
                <p
                  class="apply-job-content-1"
                >
                  4. Become the expert on all aspects of Edututs+'s data reporting and analytics infrastructure
                </p>
                <p
                  class="apply-job-content-1"
                >
                  5. Develop and maintain data definitions for internally available data sources
                </p>
              </div>
              <div
                class="apply-job-heading-41"
              >
                Who should apply?
              </div>
              <div
                class="apply-job-content-container"
              >
                <p
                  class="apply-job-content"
                >
                  Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.
                </p>
              </div>
              <div
                class="apply-job-heading-41"
              >
                What next?
              </div>
              <div
                class="apply-job-content-container"
              >
                <p
                  class="apply-job-content"
                >
                  Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.
                </p>
              </div>
            </div>
            <div
              class="apply-job-submit-application"
            >
              <div
                class="apply-job-heading-41"
              >
                Submit Your Application
              </div>
              <div
                class="apply-job-form"
              >
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="fullname"
                    name="fullname"
                    placeholder="Full Name"
                    type="text"
                  />
                </div>
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="email"
                    name="email"
                    placeholder="Email Adress"
                    type="email"
                  />
                </div>
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="phonenumber"
                    name="phonenumber"
                    placeholder="Phone Number"
                    type="number"
                  />
                </div>
                <div
                  class="ant-select ant-select-outlined apply-job-select css-dev-only-do-not-override-f7vrd6 ant-select-single ant-select-show-arrow"
                  style="color: lightgray;"
                >
                  <div
                    class="ant-select-selector"
                  >
                    <span
                      class="ant-select-selection-search"
                    >
                      <input
                        aria-autocomplete="list"
                        aria-controls="rc_select_TEST_OR_SSR_list"
                        aria-expanded="false"
                        aria-haspopup="listbox"
                        aria-owns="rc_select_TEST_OR_SSR_list"
                        autocomplete="off"
                        class="ant-select-selection-search-input"
                        id="rc_select_TEST_OR_SSR"
                        readonly=""
                        role="combobox"
                        style="opacity: 0;"
                        type="search"
                        unselectable="on"
                        value=""
                      />
                    </span>
                    <span
                      class="ant-select-selection-placeholder"
                    >
                      Gender
                    </span>
                  </div>
                  <span
                    aria-hidden="true"
                    class="ant-select-arrow"
                    style="user-select: none;"
                    unselectable="on"
                  >
                    <span
                      aria-label="down"
                      class="anticon anticon-down ant-select-suffix"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="down"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                        />
                      </svg>
                    </span>
                  </span>
                </div>
              </div>
              <div
                class="apply-job-heading-42"
              >
                Resume/CV*
              </div>
              <div
                class="apply-job-cv-form"
              >
                <input
                  accept="application/pdf"
                  name="resume"
                  placeholder="Attach Resume/CV"
                  type="file"
                />
                <div
                  class="apply-job-drap-text"
                >
                  <svg
                    class="apply-job-icon-upload"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M19.35 10.04A7.49 7.49 0 0 0 12 4C9.11 4 6.6 5.64 5.35 8.04A5.994 5.994 0 0 0 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96zM14 13v4h-4v-4H7l5-5 5 5h-3z"
                    />
                  </svg>
                  <h4>
                    Attach Resume/CV
                  </h4>
                </div>
              </div>
            </div>
            <div
              class="apply-job-links"
            >
              <div
                class="apply-job-heading-41"
              >
                Links
              </div>
              <div
                class="apply-job-form"
              >
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="linkedin"
                    name="linkedin"
                    placeholder="LinkedIn URL"
                    type="url"
                  />
                </div>
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="twitter"
                    name="twitter"
                    placeholder="Twitter URL"
                    type="url"
                  />
                </div>
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="postfolio"
                    name="postfolio"
                    placeholder="Postfolio URL"
                    type="url"
                  />
                </div>
                <div
                  class="apply-job-border-form"
                >
                  <input
                    id="otherweb"
                    name="otherweb"
                    placeholder="Other Website URL"
                    type="url"
                  />
                </div>
              </div>
            </div>
            <div
              class="apply-job-additional-infor"
            >
              <div
                class="apply-job-heading-41"
              >
                Links
              </div>
              <div
                class="apply-job-border-form"
              >
                <textarea
                  cols="10"
                  id="infor"
                  name="infor"
                  placeholder="Add a cover letter or anything else you want to share"
                  rows="10"
                />
              </div>
              <button
                class="apply-job-submit-button"
                type="submit"
              >
                Submit Application
              </button>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                ▼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          © 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
